platform/ios/src/MGLMapView.mm

```diff
@@ -1968,20 +1968,21 @@ - (void)handleTwoFingerDragGesture:(UIPanGestureRecognizer *)twoFingerDrag
     [self cancelTransitions];
 
     self.cameraChangeReasonBitmask |= MGLCameraChangeReasonGestureTilt;
-
+    static CGFloat initialPitch;
```

Comment:
How does this affect the gesture when deallocating `MGLMapView` and then allocating a new one? (i.e. `initialPitch` will have the previous view's value)

Reply:
I don’t think so, as the initial pitch is always set in the beginning state of the gesture recognizer. So it would only be a problem, if the method gets called without being in the beginning state at the first call. I think due to the UIGestureRecognizer architecture this is not possible.

Reply:
It shouldn’t be possible for a user to perform a tilt gesture on a new map view and get a stale value for `initialPitch` — `initialPitch` is set on `UIGestureRecognizerStateBegan`, which should always happen before we need this value to calculate the new pitch (to change the camera, and so on).

Reply:
Works for me - thanks @flashspys!